client/src/templates/Introduction/components/block.tsx

```diff
@@ -128,18 +129,14 @@ class Block extends Component<BlockProps> {
       );
     });
 
-    const blockIntroObj: { title?: string; intro: string[] } = t(
-      `intro:${superBlock}.blocks.${blockDashedName}`
-    );
-    const blockTitle = blockIntroObj ? blockIntroObj.title : null;
-    const blockIntroArr = blockIntroObj ? blockIntroObj.intro : [];
-    const {
-      expand: expandText,
-      collapse: collapseText
-    }: {
-      expand: string;
-      collapse: string;
-    } = t('intro:misc-text');
+    const blockTitle = [
+      t(`intro:${superBlock}.blocks.${blockDashedName}.title`)
+    ];
```

Comment:
Why is this an array?

Reply:
There is a function that require it to be an array, refactoring that function lead to creating a subcomponent, but I didn't want the PR to stay for a week, then asked to make the changes in the other PR, so I committed worse code.

I don't think there is a reason for it be an array anymore, except me not wanting to create another PR with the same changes later, and I will wait for this to be merged instead.

For context https://github.com/freeCodeCamp/freeCodeCamp/pull/49820/commits/32ee772e6c24c665a04e805747e0025ba5f9d2c3

Reply:
Or different answer, I was tunnel visioned again 🤦